akanda/rug/api/configuration.py

```diff
@@ -179,7 +179,8 @@ def fmt(fixed):
 def _subnet_config(subnet):
     return {
         'cidr': str(subnet.cidr),
-        'dhcp_enabled': subnet.enable_dhcp,
+        'dhcp_enabled': subnet.enable_dhcp and getattr(
```

Comment:
This is missing the corresponding changes in:

https://github.com/akanda/akanda-rug/blob/stable/juno/akanda/rug/api/quantum.py#L134

and in:

https://github.com/akanda/akanda-rug/blob/stable/juno/akanda/rug/api/quantum.py#L161